jumanji/training/networks/rware/actor_critic.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence
+
+import chex
+import haiku as hk
+import jax.numpy as jnp
+import numpy as np
+
+from jumanji.environments import Rware
+from jumanji.environments.routing.rware.types import Observation
+from jumanji.training.networks.actor_critic import (
+    ActorCriticNetworks,
+    FeedForwardNetwork,
+)
+from jumanji.training.networks.parametric_distribution import (
+    MultiCategoricalParametricDistribution,
+)
+from jumanji.training.networks.transformer_block import TransformerBlock
+
+
+def make_actor_critic_networks_rware(
+    rware: Rware,
+    policy_layers: Sequence[int],
+    value_layers: Sequence[int],
```

Comment:
It seems that these two are not used and that `transformer_mlp_units` is used instead. It is fine this way, but could we remove occurrences of these then? Including in the configs.
```suggestion
```